wordssite/wordssite/settings.py

```diff
@@ -106,7 +107,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Europe/Vilnius'
```

Comment:
isn't the plan to go to international waters? :)

Reply:
I can change it to UTC,there is a way of config the time zone according to the User's one?

Reply:
https://docs.djangoproject.com/en/4.0/topics/i18n/timezones/
Oh This is the UTC - got it